share/wake/lib/system/gcc.wake

```diff
@@ -27,13 +27,17 @@ def staticLFlags  = ("-flto", "-static", "-s", Nil)
 def doCompileC variant gcc flags headers cfile =
   def obj = replace `\.c(pp)?$` ".{variant}.o" cfile.getPathName
   def cmdline = gcc, distinctBy scmp flags ++ ("-c", cfile.getPathName, "-frandom-seed={obj}", "-o", obj, Nil)
-  def result = job cmdline (cfile, headers)
-  result.getJobOutput
+  makePlan cmdline (cfile, headers)
+  | setPlanLabel "compile {replace `^.*/` '' gcc} {obj}"
```

Comment:
I think I would prefer this to be named `compileC` or `compile C` just to differentiate it from other compile steps that could be invoked in a build graph.

Reply:
It is called 'compile c++ foo.o' or 'compile c foo.o' already.

Reply:
Ah, I didn't carefully read the code to understand what `gcc` was.